frontend/src/Editor/Components/Table/Table.jsx

```diff
@@ -487,6 +508,13 @@ export function Table({
                         fireEvent('onRowClicked');
                       });
                     }}
+                    onMouseOver={(e) => {
+                      const hoveredRowDetails = { hoveredRowId: row.id, hoveredRow: row.original };
+                      setRowDetails(hoveredRowDetails);
+                    }}
+                    onMouseLeave={(e) => {
+                      setRowDetails({ hoveredRowId: '', hoveredRow: '' });
+                    }}
```

Comment:
We're setting the state every time when we hover and leave the row irrespective of the `onRowHovered` event on the table row. This operation seems expensive as it will re-render the Table component. Is it possible to restrict and trigger only when we set the `onRowHovered` event?
cc: @sherfin94